cmd/mrmodelfield.cpp

```diff
@@ -1,131 +0,0 @@
-/* Copyright (c) 2008-2017 the MRtrix3 contributors.
```

Comment:
Note that, if needed, this can now be done in practice with `mtlognorm` (yet with the new, correct log-domain computations), since `mtlognorm` now also accepts a single "tissue type" (or any other image to be modelled).